DennisNjeri/SendIT
# Ft signup page #161582802

**What does this PR do?**
This PR creates a module for user registration

Description of Task to be completed?
User should be able to login through this module using either email and password or username and password. Also provided a landing page for the user.

How should this be manually tested?
Clone this repository and then run file index.html

Any background context you want to provide?
None

What are the relevant pivotal tracker stories?
A user should be able login to the system. [##161598212]
